### PR TITLE
[FEATURE] Add aws test on PRIDE data.

### PR DESCRIPTION
**Pull request by jpfeuffer**
this adds another GitHub action to test on bigger data from PRIDE.

### CODE REVIEW

**Comment by Zethson** [comment]
Awesome!
How long do you expect it to run? How big is the data?

**Comment by jpfeuffer** [comment]
You can check the exact filesize with the links given in the config, but it is roughly 1GB per RAW file -> 6GB

On a local compute server it takes about 2.5 hours I think.

**Comment by jpfeuffer** [comment]
This is the minimum for a sensible experimental design. We can go larger though if required ;)

File: .github/workflows/awstest.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: nf-core AWS test
+# This workflow is triggered on PRs to the master branch.
+# It runs the -profile 'test_full' on AWS batch
+
+on:
+  pull_request:
```
Comment (ggabernet) [inline .github/workflows/awstest.yml]:
```suggestion
  push:
```

Comment (ggabernet) [reply .github/workflows/awstest.yml]: We decided to go for push, as tests would be triggered multiple times if pushes are added to PRs to master. We want to test only once all the other tests pass.

File: .github/workflows/awstest.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    branches:
+      - master
+      - dev #TODO remove after testing
```
Comment (ggabernet) [inline .github/workflows/awstest.yml]: Once the test is triggered on AWS, please remove the `dev` branch from here 😄

**Comment by drpatelh** [comment]
You will also need to add the `test_full` profile to `nextflow.config` :wink:

File: .github/workflows/awstest.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+          python-version: 3.7
+      - name: Install awscli
+        run: conda install -c conda-forge awscli
+      - name: Start AWS batch job
+        env:
+          AWS_ACCESS_KEY_ID: ${{secrets.AWS_KEY_ID}}
+          AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_KEY_SECRET}}
+          TOWER_ACCESS_TOKEN: ${{secrets.TOWER_ACCESS_TOKEN}}
+        run: |
+          aws batch submit-job --region eu-west-1 --job-name nf-core-atacseq --job-queue 'default-8b3836e0-5eda-11ea-96e5-0a2c3f6a2a32' --job-definition nextflow --container-overrides '{"command": ["nf-core/proteomicslfq", "-r '"${GITHUB_SHA}"' -profile test_full --outdir s3://nf-core-awsmegatests/proteomicslfq/results-'"${GITHUB_SHA}"' -w s3://nf-core-awsmegatests/atacseq/work-'"${GITHUB_SHA}"' -with-tower"], "environment": [{"name": "TOWER_ACCESS_TOKEN", "value": "'"$TOWER_ACCESS_TOKEN"'"}]}'
```
[the remark this answers is not in the document]
Comment (jpfeuffer) [reply .github/workflows/awstest.yml]: oops should have used a full find/replace

File: .github/workflows/awstest.yml (hunk 1 above)
Comment (jpfeuffer) [reply .github/workflows/awstest.yml]: sure

**Comment by ggabernet** [comment]
your secrets are all set up now!

**Pull request merged**